src/dmclock_server.h

```diff
@@ -1301,13 +1310,15 @@ namespace crimson {
       // pull convenience constructor
       PullPriorityQueue(typename super::ClientInfoFunc _client_info_f,
 			AtLimitParam at_limit_param = AtLimit::Wait,
-			double _anticipation_timeout = 0.0) :
+			double _anticipation_timeout = 0.0,
+			Counter _erase_max = 100) :
 	PullPriorityQueue(_client_info_f,
-			  std::chrono::minutes(10),
-			  std::chrono::minutes(15),
-			  std::chrono::minutes(6),
+			  std::chrono::seconds(300),
```

Comment:
I know I'm responsible for this, but I'd prefer these constants not being scattered through the code and even repeated between PullPriorityQueue and PushPriorityQueue. What if we were to add some constexpr declarations at the top of this file (e.g., standard_idle_age, standard_erase_age, standard_check_time, and standard_erase_max) and use those below? Would you be alright with that?

Reply:
yes, I think it's quite reasonable to add constexpr declararions.